### PR TITLE
feat(svg): update crown logo and add missing favicon

### PR DESCRIPTION
**Pull request by sanakhandvsa**
## Description

Ticket : [VOL-4960](https://dvsa.atlassian.net/browse/VOL-4960)

update crown logo and add missing favicon

Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <meta name="theme-color" content="blue" /> 
     
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="shortcut icon" sizes="16x16 32x32 48x48" href="/assets/images/favicon.ico" type="image/x-icon">
```
Comment (JoshuaLicense) [inline index.html]: The `favicon.ico` that is part of the PR does not have `16x16` & `32x32` size, does it?

**Review by JoshuaLicense — CHANGES_REQUESTED** [review]

**Comment by JoshuaLicense** [comment]
Please add ticket number to the PR description please @sanakhandvsa

File: index.html (hunk 1 above)
Comment (sanakhandvsa) [reply index.html]: I noticed that in our repositories (`olcs-selfserve` and`olcs-internal`), we have different favicon sizes specified in the HTML as well. That's why I copied over same in `maintenence-page`

However, according to the [GOV.UK_Frontend v](https://github.com/alphagov/govuk-frontend/releases/tag/v4.8.0) 4.8.0 repo, only one 16x16 favicon is provided.  

To maintain consistency across our projects, what do you think about updating the favicon across all repositories to match the guidelines provided by GOV.UK_Frontend v4.8.0 or just remove on this repo?

Comment (JoshuaLicense) [reply index.html]: > However, according to the [GOV.UK_Frontend v](https://github.com/alphagov/govuk-frontend/releases/tag/v4.8.0) 4.8.0 repo, only one 16x16 favicon is provided.

Where is 16x16 coming from? The `favicon.ico` that is included in this PR (and the assets provided by GDS) is `48x48`. An `ico` is not a scalable format. GOV.UK provide an SVG as a scalable `favicon.ico`. For this PR, that isn't necessary.  

image

Straight from the GDS docs ⬆️ The line below shows how to provide a scalable favicon in SVG - but again, for this page, not necessary.

Documentation: https://design-system.service.gov.uk/styles/page-template/

> To maintain consistency across our projects, what do you think about updating the favicon across all repositories to match the guidelines provided by GOV.UK_Frontend v4.8.0 or just remove on this repo?

It'll be something that is refactored in the future. There's no direct link between this page and the application itself.

Comment (sanakhandvsa) [reply index.html]: Thanks changed to 48x48.

**Pull request merged**